extension.js

```diff
@@ -22,6 +24,7 @@ const Logger       = Me.imports.common.Logger;
 const Convenience  = Me.imports.convenience;
 const EXTENSIONDIR = Me.dir.get_path ();
 const APP_PATH     = EXTENSIONDIR + "/cpufreq-application";
+const IS_3_XX_SHELL_VERSION = Config.PACKAGE_VERSION.startsWith("3");
```

Comment:
The recommended way to check the major version is to perform a split on `PACKAGE_VERSION` and get the first element ([source](https://gjs.guide/extensions/upgrading/gnome-shell-40.html#metadata-json)). I know it's a long shot, but using `startsWith` can lead to a false positive on version numbered after 300. Let's not repeat the infamous "Windows 9" error.

Reply:
We are at version 42 of gnome shell, a new version is released every 6 months.
To get to version 300, one would have to wait (300-42) / 2 = 129 years. It could be assumed that 3_XX versions of gnome shell will no longer be supported in 129 years